src/spatial_reaction_systems/spatial_ODE_systems.jl

```diff
@@ -0,0 +1,211 @@
+### Spatial ODE Functor Structures ###
+
+# Functor structure containg the information for the forcing function of a spatial ODE with spatial movement on a lattice.
+struct LatticeDiffusionODEf{R,S,T}
+    ofunc::R
+    nV::Int64
+    nS::Int64
+    pV::Vector{Vector{Float64}}
```

Comment:
Can you use somewhat more descriptive names for others to more easily understand what these fields represent? I assume `nV` is the number of vertices? Maybe use `numverts`, `numspecs`, etc? This is also not really conforming to Julia naming conventions with the mixed case. Finally, can you add doc strings of what these fields are representing?

Reply:
If `pV` are the vertex parameters, you could use `vertps` or `vert_ps` as MTK uses `ps`.

Reply:
fixed